EventsExpress/Controllers/EventController.cs

```diff
@@ -128,6 +128,26 @@ public async Task<IActionResult> AddUserToEvent(Guid userId, Guid eventId)
             return BadRequest();
         }
 
+        /// <summary>
+        /// This method have to add user to category.
+        /// </summary>
+        /// <param name="userId">Required.</param>
+        /// <param name="eventId">EventId.</param>
+        /// <param name="action">button press.</param>
+        /// <response code="200">Approving user from event process success.</response>
+        /// <response code="400">If aproving user from event process failed.</response>
+        [HttpPost("[action]")]
+        public async Task<ActionResult> ApproveUserToEvent(Guid userId, Guid eventId, bool action)
```

Comment:
Let's create explicit `ApproveVisitor` and `DenyVisitor` endpoints and call service with appropriate parameters.